sdk/keyvault/Azure.ResourceManager.KeyVault/samples/README.md

```diff
@@ -0,0 +1,12 @@
+---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure-resource-manager
+name: Azure.ResourceManager.KeyVault samples for .NET
+description: Samples for the Azure.ResourceManager.KeyVault client library
+---
+
+# Azure.ResourceManager.KeyVault Samples
```

Comment:
I saw you removed the link to the sample as it failed the CI link verification. Can we just use a relative link like:
`- [Managing Key Vaults](Sample1_ManagingKeyVaults.md)`

Reply:
Relative link will get this error https://dev.azure.com/azure-sdk/public/_build/results?buildId=1092901&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=72c41882-9d21-5580-4869-7f3ea1bd9c0a&l=18